shangguanwang/NFT-marketplace
# Add Favicon

## Description
Add favicon

## Related Issue
closes #28 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Testing Steps / QA Criteria
- From your terminal, pull down this branch with `git pull origin visual-effect` and check that branch out with `git checkout visual-effect`
- Run `npm run dev` to launch the app
- You should see the Favicon
